project/Dependencies.scala

```diff
@@ -34,8 +34,8 @@ object Dependencies {
     val commonsText      = "1.10.0"
     val jodaTime         = "2.10.1"
     val useragent        = "1.21"
-    val uaParser         = "1.4.3"
-    val snakeYaml        = "1.31" // override transitive dependency to mitigate security vulnerabilities CVE-2022-25857
+    val uaParser         = "1.5.4"
+    val snakeYaml        = "2.0"
```

Comment:
If I run `dependencyTree` in sbt then I see this:

```
[info]   +-com.github.ua-parser:uap-java:1.5.4
[info]   | +-org.apache.commons:commons-collections4:4.4
[info]   | +-org.yaml:snakeyaml:1.33 (evicted by: 2.0)
[info]   | +-org.yaml:snakeyaml:2.0
[info]   | 
```

Is this deliberate to evict 1.33 with a new major version, and are you sure there is no binary incompatibility?

Reply:
I also don't like that. I did it like in [this OSS contribution](https://github.com/snowplow/enrich/issues/761).

We do have unit tests that use this library and they pass, but I don't know if the eviction happens for the unit tests or only when we build the artifact. do you know ?

Let me check the vulnerabilities with `1.33`.

Reply:
> I don't know if the eviction happens for the unit tests or only when we build the artifact. do you know ?

Eviction happens for unit tests too.  So it was probably OK.

Even so, if you're finding that 1.33 has no vulnerabilities, then it seems safer to use 1.33.

Thanks for checking!